mizar/obj/tests/test_net1.yaml

```diff
@@ -22,10 +22,10 @@
 apiVersion: "mizar.com/v1"
 kind: Subnet
 metadata:
-  name: net1
+  name: subnet1
 spec:
   vpc: "vpc1"
-  vni: "0"
+  vni: "15654902"
```

Comment:
Not sure if the vpc whose vni is 15654902 DO exist when we apply test-net1.yaml

Reply:
I have a similar question here. if the vpc1 that's already created in the cluster has vni "3456", do we put "3456" here? what does this vni do?

Reply:
and it goes back to the original question, which is the unique identifier of a vpc? is it name or vni. both options are provided here. what happens if the provided vni here is different from the actual vni of the vpc1?